components/console/helpers/formatterhelper.rst

```diff
@@ -63,3 +63,64 @@ messages and 2 spaces on the left and right).
 The exact "style" you use in the block is up to you. In this case, you're using
 the pre-defined ``error`` style, but there are other styles, or you can create
 your own. See :ref:`components-console-coloring`.
+
+Print Truncated Messages
+------------------------
+
+.. versionadded:: 3.1
+    The ``truncate`` method was introduced in Symfony 3.1.
+
+Sometimes you want to print a message truncated to an explicit character length. This is possible with the
```

Comment:
I know it's a minor thing, but we require it for all the documentation: contents should be wrapped around the column 80. Otherwise sentences are too long to read. Thanks!